pkg/rbd/rbd_util.go

```diff
@@ -29,7 +29,6 @@ import (
 	"time"
 
 	"github.com/ceph/ceph-csi/pkg/util"
-
```

Comment:
This empty line is intentional. Split between local `pkg` and other external dependencies.

Reply:
Brought back.